### ISSUE TITLE
Assembly not filtering out contigs less than 300 bp when default is used

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#2417]
This should be addressed

**Comment by olsonanl** [comment]
do you have an example where you see this? I'd thought this was in place.

**Comment by ARWattam** [comment]
Yes.  Have showed it to Allan.  I would say that any of my recent assembly jobs have contigs less than 300bp.  He plans on fixing it when he is done with Primer3


From: olsonanl ***@***.***>
Date: Tuesday, July 13, 2021 at 2:03 PM
To: PATRIC3/patric3_website ***@***.***>
Cc: Wattam, Rebecca (arw3s) ***@***.***>, Author ***@***.***>
Subject: Re: [PATRIC3/patric3_website] Assembly not filtering out contigs less than 300 bp when default is used (#2417)

do you have an example where you see this? I'd thought this was in place.
